smoke-tests/fake-backend/build.gradle

```diff
@@ -28,7 +26,70 @@ dependencies {
   implementation("org.slf4j:slf4j-simple:1.7.30")
 }
 
+shadowJar {
+  manifest {
+    attributes 'Main-Class': 'io.opentelemetry.smoketest.fakebackend.FakeBackendMain'
+  }
+}
+
+ext {
+  extraTag = findProperty("extraTag") ?: new Date().format("yyyyMMdd.HHmmSS")
+}
+
 jib {
   from.image = "gcr.io/distroless/java-debian10:11"
-  to.image = "ghcr.io/open-telemetry/java-test-containers:smoke-fake-backend-${new Date().format("yyyyMMdd.HHmmSS")}"
+  to.image = "ghcr.io/open-telemetry/java-test-containers:smoke-fake-backend-$extraTag"
 }
+
+
+def backendDockerBuildDir = new File(project.buildDir, "docker-backend")
```

Comment:
```suggestion
# windows containers are built manually since jib does not support windows containers yet

def backendDockerBuildDir = new File(project.buildDir, "docker-backend")
```